contrib/cirrus/postbuild.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# This script attempts to confirm all included go modules from
+# other sources match what is expected in `vendor/modules.txt`
+# vs `go.mod`.  Also make sure that the generated bindings in
+# `pkg/bindings/...` are in sync with the code.  It's intended
+# for use after successfully building podman, to prevent wasting
+# time on tests that might otherwise succeed with bad/ugly/invalid
+# code.
+
+source /etc/automation_environment
+source $AUTOMATION_LIB_PATH/common_lib.sh
+
+# There's no need to perform further checks on more than one
+# CI platform.  These variables are defined in .cirrus.yml
+# shellcheck disable=SC2154
+if [[ ! "${DISTRO_NV}" =~ ${FEDORA_NAME} ]]; then
+    echo "Skipping additional checks on $DISTRO_NV"
+    exit 0
+fi
```

Comment:
I'm not so sure this `skip` is a good idea. Did you look at the logs? Did you see the humongous diff between f36 [intel](https://cirrus-ci.com/task/5841637799624704?logs=postbuild#L13) and [aarch64](https://cirrus-ci.com/task/6404587753046016?logs=postbuild#L13)? Given the sometimes significant differences in `go` between distros, I think we would be wise to run this check everywhere.

Reply:
okay, fair enough.

Reply:
Ugh...[I was just reminded why I added this](https://cirrus-ci.com/task/5070067325992960).  We need to exclude Ubuntu.